frontend/kesaseteli/employer/src/components/application/steps/step3/Step3Summary.tsx

```diff
@@ -24,7 +25,26 @@ const Step3Summary: React.FC = () => {
     <ApplicationForm stepTitle={t('common:application.step3.header')}>
       <ApplicationSummary />
       <FormSection columns={1}>
-        <Checkbox id="termsAndConditions" validation={{ required: true }} />
+        <Checkbox
+          id="termsAndConditions"
+          validation={{ required: true }}
+          label={
+            <Trans
```

Comment:
The text content in finnish might be confusing first but it's default fallback which is needed for jest tests. Application really never uses the default text as it reads the translations from common.json

https://react.i18next.com/latest/trans-component